Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -0,0 +1,269 @@
+# AEP Analytics & Edge Bridge
+## Overview
+This hands-on tutorial provides end-to-end instructions for Mobile SDK customers on how  Edge Bridge can help easily migrate from AEPAnalytics to AEP.
+
+There are other beginning states the customer can be in like using ACP Analytics, v4 extension, etc. These usages can potentially be migrated to AEP and from there, this tutorial can be applied afterwards. However, this should be determined on a case-by-case basis, as for example in the v4 extension case, it may be more effective to simply implement AEP extensions itself without the need for Edge Bridge migration path.
+
+Tutorial setup:
+
+Initial test app that is based on prerequisites (implements Analytics)
+Final test app after all the tutorial steps have been implemented (implements EdgeBridge)
+
+Tutorial steps:
+
+      0. Prerequisites (set up Analytics report suite, mobile property, Assurance). List out required permissions for this tutorial: Analytics, Schema creation, Data Collection (Launch tags), Datastream view and edit, Assurance.
+
+## Data Collection config instructions
+Create XDM schema   
+Configure Datastream, enable Analytics - 2 paths:   
+    - same rsid(s) as in Analytics extension (if using Analytics + EdgeBridge this will cause double counting).  
+    - different rsid(s) if the customer wants to start new or run the migration in a comparison mode (Analytics + EdgeBridge side by side).  
+
+Install Edge Network & Edge Identity in Launch - Edge Bridge does not have a card here  
+Analytics should remain installed in Launch for production app versions.  
+Publish the changes  
+
+## Client-side implementation
+### 1. Install Edge Bridge using dependency manager
+To install EdgeBridge, use the currently the supported installation options:
+### iOS
+#### [CocoaPods](https://guides.cocoapods.org/using/using-cocoapods.html)
+
```

Comment:
In this section include specific step-by-step instructions. This will be easier once the tutorial app is done.

1. open Podfile
2. edit Podfile as the following: ...
3. from a terminal window, run `pod update`